docs/FAQs.md

```diff
@@ -169,3 +169,10 @@ A mar file can be used either locally or be publicly available via http. An S3 U
 
 ### How to set a model's batch size on SageMaker?  Key parameters for TorchServe performance tuning.
 [TorchServe performance tuning example](https://github.com/lxning/torchserve_perf/blob/master/torchserve_perf.ipynb)
+
+## Why is my model initialization so slow?
+There's a few reasons why model initialization can be slow
+1. `torch.load()` overhead - not something we can improve, this will be more dramatic for larger models
```

Comment:
Is the time taken without torchserve (out-of-box pytorch model load) --vs-- with torchserve similar? I wonder if there's other sources of overhead induced by torchserve only.

Reply:
3 is the main one I've seen trip people up